src/PersonObjects/Player/PlayerObjectGeneralMethods.tsx

```diff
@@ -1870,13 +1870,12 @@ export function getNextCompanyPosition(
 
 export function quitJob(this: IPlayer, company: string): void {
   if (this.isWorking == true && this.workType.includes("Working for Company") && this.companyName == company) {
-    this.isWorking = false;
-    this.companyName = "";
+    this.finishWork(true);
   }
+  delete this.jobs[company];
```

Comment:
hmm would this blast multiple jobs for the same company? i guess thats not possible right?